src/ObjectGraph.es6.js

```diff
@@ -74,6 +74,11 @@ ObjectGraph.prototype.blacklistedObjects = [
 }.bind(typeof window === 'undefined' ? global : window)).filter(
   value => value !== undefined
 );
+ObjectGraph.prototype.blacklistedTypes = [
+  'MimeType'   // MimeType Object initiates a process of continually
+               // extending the object graphs in Chrome
```

Comment:
What's the largest version number where this issue occurs? I think we should include it in the comment.

nit: Still missing period at end of comment.